src/__tests__/planning.test.ts

```diff
@@ -1,5 +1,5 @@
 import {Plan, plan, Device, AxidrawFast, XYMotion, PenMotion, defaultPlanOptions} from '../planning';
-import {Vec2} from '../vec';
+import type {Vec2} from '../vec';
```

Comment:
No space between `{` `}`, but I expect you're still experimenting with the tool?

Reply:
Yeah, for now in `biome.json` the linter is enabled but the formatter is not.

I just tried enabling the formatter (with default settings), and it did change to `{ Vec }`.... but also expanded all the imports on different lines. :/